### ISSUE TITLE
Error code -65541

### ISSUE THREAD

**Issue by rdamazio** [synergy/synergy#4375]
Running synergy 1.6.2 as a client on OSX 10.10.2 (server is 1.6.2 on Linux), I get this error:

![screen shot 2015-02-20 at 14 22 09](https://cloud.githubusercontent.com/assets/997641/6310485/3695a948-b90c-11e4-943f-1ba8345f5297.png)

This happens specially after synergy has disconnected (e.g. because the laptop it runs on went to sleep then came back).
The only solution is to manually kill synergyc on the command line (exiting the Synergy app doesn't kill it).

**Comment by nbolton** [comment]
Yeah I've seen this too on my MacBook Air, but not on my desktop so far. Prioritized.

**Comment by nbolton** [comment]
Perhaps related or same as #4303?

**Comment by totolipton** [comment]
I have the same error, using synergy on macbook and windows. It's really annoying.

**Comment by szines** [comment]
Same error on Mac Book Pro 15. Server is Mac OSX 10.10.2. Client could be windows or linux... Synergy works properly, but if you try to click on the icon in the task bar, all menu point is disabled. If you try to launch Synergy from Launchpad, it does nothing for 10 seconds, but after appear the application window with this Error code: -65541 popup.

**Comment by carloshgaray** [comment]
Same error here --> MBP retina 15 10.10.3 (14D136)

**Comment by robkooper** [comment]
I just had this happen on my macbook, running. This seems to be related to airplay. I got the following error in my console log.

```
6/18/15 9:04:03.075 AM coreaudiod[203]: 2015-06-18 09:04:03.075453 AM [AirPlay] ### Write audio (3528 bytes) when client not fully started
```

**Comment by thanso** [comment]
I am getting this error as well. Mac Pro desktop. HD sleep set to 'never'.

Mac #1 (server) - OS 10.14.6 / Barrier 2.3.3
Mac #2 (slave) - OS 10.12.6 / Barrier 2.3.0

All works perfectly for about 45 mins, then suddenly get looping error window. The connection between the two Mac continues, but the onscreen error dialog just keeps appearing and visually blocks every open window.
Screen Shot 2021-03-19 at 12 32 25 PM

Only way to resolve is to force quit. This happens whether the 2nd Mac is powered on or not.

I was formally using Synergy to share the keyboard/mouse and the same looping error occurred.
